concordium-std/src/impls.rs

```diff
@@ -1,7 +1,19 @@
 use crate::{convert, mem, num, prims, prims::*, traits::*, types::*};
+#[cfg(not(feature = "std"))]
+use alloc::{
+    collections::{BTreeMap, BTreeSet},
+    string::String,
+    vec::Vec,
+};
```

Comment:
This should not be here we already have conditional re-exports in lib.rs, so there is no need for the conditional imports here. These can just be added to `use crate:: ...`.

In general conditional imports should be limited as much as possible to lib.rs.